ckanext/security/logic/auth.py

```diff
@@ -0,0 +1,15 @@
+'''
+Action functions, all sysadmin-only
+'''
+
+def security_throttle_user_reset(context, data_dict):
+    return {'success': False}
```

Comment:
Hi @wardi, thanks for your contribution here... just wondering why you've defined these at all if they have no implementation? Is this just a plugin best-practice sort of thing?

Reply:
This auth function is called here: https://github.com/data-govt-nz/ckanext-security/pull/43/files#diff-49d7798805a3c5f89df30f1ab7b4f9cddc01b94da3712b0071fcb7b2c3c22c4fR21

Using auth functions and `check_access` is a best practice for API actions because it allows other plugins to override auth checks for these API endpoints.